code/__HELPERS/global_lists.dm

```diff
@@ -60,8 +60,6 @@
 
 	init_crafting_recipes(GLOB.crafting_recipes)
 
-	init_subtypes_w_path_keys(/obj/projectile, GLOB.proj_by_path_key)
```

Comment:
What was this ?

Reply:
code used by the gun examine code that create every projectile for some reason, this was removed and replaced with initial() as it was stupid and broke with stuff like the organ projectile